### PR TITLE
Initial setup

### LINKED ISSUE #1: Bring reusable content from information-2023 here
Start building out this repository

### PR DESCRIPTION
**Pull request by palbee**
This is the starting point for the 2024 information repo.

Closes #1 

In the spirit of minimizing surprises, all commits must be reviewed.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
The date of hackathon is October 19-20.

**Comment by palbee** [comment]
This is what I get for editing before 6am.

**Comment by palbee** [comment]
Now with corrected dates.

**Comment by palbee** [comment]
@arun-gupta Could you button mash this?

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+## What is the hackathon timeline?
+
+TBD: Application open  
+Sep 6: Application close  
+Sep 20: Teams announced  
+Oct 19: Hackathon start  
+Oct 20:  Hackathon end  
+
+## How can I participate in the hackathon?
+Register your team at https://ai-event.ted.com/hackathon.  We will then evaluate their submission and let them know. The space is limited so don't wait.
```
Comment (arun-gupta) [inline faq.md]: Hackathon page is https://tedai-sanfrancisco.ted.com/hackathon/

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+If you have any further questions or need additional information, please do not hesitate to reach out to the hackathon core team using tedaihackathon@xy.ai.
+
+## What data may I use?
+Data should come from public sources with Creative Commons or permissive licenses or cited according to the data creator’s/publisher’s guidance. The data used in the hackathon must be accessible to a global audience. 
+
+## Where will the code reside and what licenses will be considered?
+LF AI & Data Foundation has created a GitHub organization. We recommend all participants publish their source code as a repo to this organization. The code must use an OSI-approved license.
+
+## How do I contact organizers about the hackathon?
+Send us an email at tedaihackathon@xy.ai.
```
Comment (arun-gupta) [inline faq.md]: Change this to TEDAI.hackathon@gmail.com

File: faq.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+Frequently Asked Questions
+
+## What is TED AI Open Source Hackathon?
+The [TED AI Open Source Hackathon](https://ai-event.ted.com/hackathon) is an exciting programming event that challenges you to find ways to use Generative AI to tackle social and environmental issues. You will have the opportunity to use your coding skills and collaborate with fellow developers, all in the name of good.
```
Comment (arun-gupta) [inline faq.md]: Change the link to https://tedai-sanfrancisco.ted.com/hackathon/

**Review by arun-gupta — CHANGES_REQUESTED** [review]
Some minor changes, looks good otherwise

File: faq.md (hunk 3 above)
Comment (palbee) [reply faq.md]: Addressed in 75a5b311cac6e25ccad6dade58ace598e559553d

File: faq.md (hunk 2 above)
Comment (palbee) [reply faq.md]: Addressed in 75a5b311cac6e25ccad6dade58ace598e559553d

File: faq.md (hunk 1 above)
Comment (palbee) [reply faq.md]: 75a5b311cac6e25ccad6dade58ace598e559553d

**Comment by palbee** [comment]
Thanks for the updates.

**Review by arun-gupta — APPROVED** [review]
LGTM

**Pull request merged**